index.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+require('env2')('.env');
+
+const Promise = require('bluebird');
+const sauce = require('mintsauce');
```

Comment:
I am wondering what the added value is of using this library instead of just calling the next function with a callback?